apps/src/applab/Visualization.jsx

```diff
@@ -20,6 +20,16 @@ var styles = {
   },
   phoneFrame: {
     marginBottom: 0
+  },
+  screenBlock: {
+    backgroundColor: 'rgba(0, 0, 0, 0.05)',
```

Comment:
Started out with that, but it wasn't very noticeable on apps with a white background (part of this might just be that my test apps dont have enough going on). It is quite possible that we go back to white.